src/Main.java

```diff
@@ -1,5 +1,58 @@
+import car.Car;
+import car.CarOwner;
+import carwash.*;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        CheapWash cheapWash = new CheapWash();
+        RegularWash regularWash = new RegularWash();
+        PremiumWash premiumWash = new PremiumWash();
+        CarWashService carWashService = new CarWashService(cheapWash);
+        CarWashService carWashService2 = new CarWashService(new RegularWash());
+        CarWashService carWashService3 = new CarWashService(new PremiumWash());
```

Comment:
Use common interface `WashStrategy` when creating washes.
Create only one service and change the strategy via setter